src/pages/ProfilePage/ProfilePage.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Alert from 'react-bootstrap/Alert';
+
+import { useUserContext } from '../../contexts/UserContext';
+import { logIn, changePassword } from '../../lib/auth';
+import './ProfilePage.css';
+
+function ChangePasswordForm(): JSX.Element {
```

Comment:
I think maybe we should abstract this to its own dir in the component directory just so that we can independently test this component.

Reply:
Or at least export this so we can have a separate spec file and import it

Reply:
Yes, that's a good idea. Let me make that change.